src/units/unit.hpp

```diff
@@ -2098,6 +2098,16 @@ class unit : public std::enable_shared_from_this<unit>
 
 	bool is_fearless_, is_healthy_;
 
+	// Unit list/recall list favorite unit marking
+	bool is_favorite_;
```

Comment:
I'm dubious about making this part of `unit`. It's only relevant to the display thereof, not the unit class itself. Probably should make it part of prefs so it can be saved.

Reply:
Oh, it's *campaign* scope. Ok, that makes more sense. Alright, leave it in `unit`.